src/display/editor/text.js

```diff
@@ -269,7 +269,7 @@ class TextEditor extends AnnotationEditor {
     if (!this.isInEditMode()) {
       return;
     }
-
+    this.#setEditorDimensions();
```

Comment:
Not clear what the purpose of this change is. Remove all changes to this file if they are not needed

Reply:
Bro, This will bring back the sticky note to the edit layer. If the sticky note is empty the container won’t return to the editor layer. so by calling this function even if the sticky note is empty, it will return the container to the editor layer.

Reply:
@barath46 but bro this ticket is for the highlight annotation changes, why are we talking about the sticking notes?

Reply:
Bro, I worked on this yesterday. It is just a small change so I thought to push with this feature

Reply:
@barath46 - bro I'm not confident that this is the actual fix for the sticky note issue, let's keep changes separate

Reply:
Okay bro